packages/ef-runtime-client/src/ModuleLoader/ModuleLoader.ts

```diff
@@ -0,0 +1,29 @@
+export interface IModuleLoaderDependencies {
+  document: Document;
+  loaderSrc: string;
+}
+
+export class ModuleLoader {
+  private document: Document;
+  private loaderSrc: string;
+
+  constructor({ document, loaderSrc }: IModuleLoaderDependencies) {
+    this.document = document;
+    this.loaderSrc = loaderSrc;
+  }
+
+  async init(): Promise<void> {
```

Comment:
Of two minds if this is the right way to initialise the System runtime.  

I guess the whole "how do we include SystemJS" question informs the right way to move forward.  I think bundling may be the best way, at which point this is a much simpler method call.